python/build-layer.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+DISTRO_DIR=build
+SOURCES_DIR=otel/otel_sdk
+OTEL_PYTHON_DIR=../opentelemetry-lambda/python/src
+
+echo "Modify dependencies for Splunk integration"
+pushd "$OTEL_PYTHON_DIR/$SOURCES_DIR"
+sed -i 's/^opentelemetry-distro.*/splunk-opentelemetry[all]==1.0.0/g' requirements.txt
+popd
+
+echo "Building OTel Lambda python"
+pushd $OTEL_PYTHON_DIR
+./build.sh
+otel=$?
+
+echo "Preparing Splunk layer"
```

Comment:
does it make sense to continue if `./build.sh` failed?
if not then maybe we could change the error handling to:

```sh
if [[ $? -ne 0  ]] ; then
  echo "Building OTel Lambda python failed"; exit 1
else
```

after each script that may fail

if you want to keep the scripts short then you could

```sh
[[ $? -ne 0  ]] && echo "Building OTel Lambda python failed"; exit 1
```

BTW have you considered adding `set -eu` at the top of the script instead? things like `unzip` could fail as well I guess

_Do whatever you think would be the best 😉 _